Benqi-Agents/Qi-Transfers/README.md

```diff
@@ -0,0 +1,70 @@
+# QI Token Large Transfer and Large Balance Agent
+
+## Description
+
+This agent detects transfers of a large amount of `QI` tokens and accounts with large amount of `QI` balance.
+
+- Threshold for transferred token amount can be set at `src/utils.ts` line 17. The default amount is set to 1 million tokens.
+
+```
+const THRESHOLD_TRANSFER_AMOUNT: number = 1000000;
+```
+
+- Threshold percentage of `QI` token total supply for balances can be set `src/utils.ts` line 16. The default value is set to 5 percent.
+
+```
+const THRESHOLD_BALANCE_PERCENTAGE: number = 5;
+```
+
+## Supported Chains
+
+- Avalanche
+
+## Alerts
+
+- BENQI-3-1
+
+  - Fired when the amount of transferred `QI` tokens are above threshold value
+  - Severity is always set to "Info"
+  - Type is always set to "Info"
+  - Metadata includes:
+    - from: Source address
+    - to: Destination address
+    - amount: The amount of transferred tokens
+
+- BENQI-3-2
+  - Fired when balance of the destination account of a transfer is more than threshold percentage of the total supply
+  - Severity is always set to "Info"
+  - Type is always set to "Info"
+  - Metadata includes:
+    - account: The account that gets the tokens
```

Comment:
here I mean that it's the account in the `toAddress` because the sender and the receiver and anyone else might hold the tokens but the agent only checks the balance of the receiver address.